index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Personal Portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <!-- The navigator row at the top -->
+    <header>
+      <a href="#"><h4>Raymond OLISA</h4></a>
+      <a href="#"><img src="img/menu-icon.png" alt = "Menu"></a>
```

Comment:
- [ ] Please replace `a` tag with `nav` tag because The `nav` tag is for semantics reason and the right position for convention and user experience for mobile devices.

Reply:
The h4 element has hover effect from the figma template